packages/frontend/components/ui/checkbox.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+
+function Checkbox({
+  name,
+  checked,
+  type,
+  disabled,
+  className,
+  value,
+  label,
+  required,
+  onChange,
+  ...otherProps
+}) {
+  let customStyleName = '';
+  if (checked && type === 'success') {
+    customStyleName = 'success-check-box';
+  } else if (checked && type === 'danger') {
+    customStyleName = 'danger-check-box';
+  }
+
+  return (
+    <div className="flex items-center">
+      <input
```

Comment:
i forgot to tell but we can use `Field from formik` instead of input. I was create our inputs with <input> tag but after start to using Formik, i see that using Field is better idea because it has own props.

for example, you can [check input component](https://github.com/Ucurtma/ucurtma-app/blob/afc78d2c558806249fd2a6a189737edd11a3757b/packages/frontend/components/ui/input.js#L34).